lib/validators/json-schema-next.js

```diff
@@ -90,6 +95,28 @@ class JsonSchemaValidator {
     }
   }
 
+  /**
+   * Resolve stringified JSON Schema to an Object.
```

Comment:
I wonder if this is the right terminology, as a JSON Schema does not have to be an object, just valid JSON as boolean schemas are permitted.